core/tauri/src/api/process/command.rs

```diff
@@ -364,14 +373,20 @@ impl Command {
           CommandEvent::Terminated(payload) => {
             code = payload.code;
           }
-          CommandEvent::Stdout(line) => {
-            stdout.push_str(line.as_str());
-            stdout.push('\n');
-          }
-          CommandEvent::Stderr(line) => {
-            stderr.push_str(line.as_str());
-            stderr.push('\n');
-          }
+          CommandEvent::Stdout(line) => match line {
+            Buffer::Text(line) => {
+              stdout.push_str(line.as_str());
+              stdout.push('\n');
+            }
+            Buffer::Raw(_) => {}
```

Comment:
@amrbashir I'd like to implement "raw" data also for `output` method. What do you think would be the best approach to that?

Reply:
you can make `stdout` and `stderr` of the `Output` struct to be `Vec<u8>` and anyone can call `String::from_utf8` on it to turn it into a string